src/main/java/com/atlassian/shele/shelePlugin/ao/IssueService.java

```diff
@@ -57,6 +55,7 @@ public void persistIssueEntity(IssueEvent event) {
         issueEntity.save();
     }
 
+    //string замаппить лист лонгов через маппер
```

Comment:
it is better to write all comments in english, also for feature implementations add todo clause